src/SapNwRfc/Internal/Interop/RfcDirection.cs

```diff
@@ -0,0 +1,14 @@
+using System.Diagnostics.CodeAnalysis;
```

Comment:
Seems unused. Can this file be removed?

Reply:
Oh right. I fiddled around with the NetWeaver SDK to read the property length and forgot to remove this.